src/components/AnnualReport/ARDate.vue

```diff
@@ -9,20 +9,12 @@
   </v-card>
 </template>
 
-<script lang="ts">
-// Libraries
-import { Component, Mixins } from 'vue-property-decorator'
-import { State, Getter } from 'vuex-class'
+<script setup lang="ts">
+import { DateComposable, AllowableActionsComposable, FilingComposable } from '@/composables'
 
-// Mixins
-import { DateMixin } from '@/mixins'
-
-@Component({})
-export default class ArDate extends Mixins(DateMixin) {
-  @State nextARDate!: string
```

Comment:
Where does this come from?

Reply:
I didn't implement this. I am not sure of the best way to implement state it seems like one of the advantages of the composition api is the ability to share states between different components. So I am not really sure what the best way is to handle ```@state```

Reply:
Accessing state directly was a cheat, left over from the initial implementation of this UI. It's much better to use getters.

Reply:
Oh okay! This could possibly be a good refactor to do before changing API's I am making some tickets for that. I can add this.